src/UnitySampleProject/Assets/Samples/web3.unity SDK/2.5.0/Web3.Unity Samples/Scripts/Scenes/SampleMain/EVM/EvmCalls.cs

```diff
@@ -158,55 +187,82 @@ public async void GetTransactionStatus()
         SampleOutputUtil.PrintResult(output, nameof(Evm), nameof(Evm.GetTransactionStatus));
     }
     
-    public async void RegisterContract()
+    /// <summary>
+    /// Uses a registered contract
+    /// </summary>
+    public async void RegisteredContract()
     {
-        var balance = await Evm.UseRegisteredContract(Web3Accessor.Web3, registeredContractName, registeredContractmethod);
+        var balance = await Evm.UseRegisteredContract(Web3Accessor.Web3, registeredContractName, EthMethod.BalanceOf);
```

Comment:
are the methods in `EthMethods` exclusive to Eth network or any Evm based network, asking if we maybe need a rename?

Reply:
evm method - ethereum virtual machine method - eth method. I think it's fine.

Reply:
@robGG1997 balance of is an ERC20 contract method. Not sure why it's listed under `EthMethod`

Reply:
because it's also a 721/155 method, we don't need to have 3 xD

Reply:
I only mention EthMethod in this case as it might be mistaken for the method you'd use to check your wallet's eth balance.